<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientDialog</class>
 <widget class="QDialog" name="ClientDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1017</width>
    <height>726</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WebSocket客户端--测试版</string>
  </property>
  <widget class="QTextEdit" name="textEdit_Log">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>270</y>
     <width>301</width>
     <height>351</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QTextEdit" name="lineEdit_Message">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>240</y>
     <width>301</width>
     <height>341</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入发送文本</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>250</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>日志：</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>911</width>
     <height>491</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="pushButton_Send">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>440</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>发送信息</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>19</y>
     <width>911</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLineEdit" name="lineEdit_ServerUrl">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>40</y>
      <width>281</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>ws://localhost:1234</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>请输入服务器以进行连接：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Connect">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>40</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>连接服务器</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Disc">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>40</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>断开服务器</string>
    </property>
   </widget>
  </widget>
  <zorder>groupBox_2</zorder>
  <zorder>groupBox</zorder>
  <zorder>textEdit_Log</zorder>
  <zorder>lineEdit_Message</zorder>
  <zorder>label_2</zorder>
  <zorder></zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
